model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/address/AddressResponseAddress.scala

```diff
@@ -96,12 +102,13 @@ object AddressResponseAddress {
       censusEstabType = other.censusEstabType,
       countryCode = other.countryCode,
       lpiLogicalStatus = lpiLogicalStatus,
-      fromSource = other.fromSource,
       confidenceScore = 100D,
-      underlyingScore = if (other.distance == 0) other.score else (other.distance/1000).toFloat
+      underlyingScore = if (other.distance == 0) other.score else (other.distance/1000).toFloat,
```

Comment:
What's the , for?

Reply:
Scala 2.12 lets you have comma on the last item. I think there was another field after the underlying score at one point.